### DISCUSSION TITLE
Is this usable without gnome?

### DISCUSSION THREAD

**Discussion by oredaze** [Schneegans/Fly-Pie#89] [Fly-Pie Q&A] [answered]
As the title says... I am not using a desktop environment and I want to try this, but I guess it's not possible?

**Comment by Schneegans** [comment, marked as answer]
The simple answer is no. A little bit longer answer is here: https://github.com/Schneegans/Fly-Pie/issues/22#issuecomment-691916530 :smile:

**Reply by oredaze** [reply]
Thank you, I will give up on this since using gnome is not an option.

**Reply by ykhurshid** [reply]
You can always use the older [gnome-pie](https://schneegans.github.io/gnome-pie.html) which I believe is usable in other window managers and desktop environments. I actually only found out about Fly-Pie entirely by accident by browsing the home page for gnome-pie.

**Reply by oredaze** [reply]
I already know about gnome-pie, but fly-pie functions differently, also it's just so cool I wanted to try it, so I went to gnome for a second... Yes I like fly-pie a whole lot, but this alone is not able to make me switch to the awful thing that is gnome.

**Comment by Schneegans** [comment]
Btw, I just published some [news regarding the platform-agnostic pie menu](https://ko-fi.com/post/Introducing-Ken-Do-L3L7L0FQ2) I am currently working on! This will not only work on Linux distributions without GNOME, but also on Windows! At the end of the post, I wrote down some ways how you could contribute to the project. I would be more than happy if you would follow the progress and support the development along the way :smile:

**Reply by oredaze** [reply]
Very exciting news. I look forward to testing it when it reaches beta.
